Templates/Integrations/DataEnrichmentThreatIntelligence/DataEnrichmentThreatIntelligence.py

```diff
@@ -0,0 +1,320 @@
+            'SSDeep': result.get('ssdeep'),
+            'Description': result.get('description')
+        }
+        # Gets DBot score
+        score = client.calculate_dbot_score(result.get('severity'))
+        # Building a score for DBot
+        dbot_score = [
+            {
+                'Indicator': result.get(hash_name),
+                'Type': 'hash',
+                'Vendor': f'{INTEGRATION_NAME}',
+                'Score': score
+            } for hash_name in FILE_HASHES if result.get(hash_name)
+        ]
+        context = {
+            outputPaths['dbotscore']: dbot_score,
+            f'{INTEGRATION_CONTEXT_NAME}(val.ID && val.ID === obj.ID)': context_entry
+        }
+
+        if score == 3:  # If file is malicious, adds a malicious entry
+            context[outputPaths['file']] = [{
+                hash_name.upper(): raw_response.get(hash_name),
+                'Malicious': {
+                    'Vendor': f'{INTEGRATION_NAME}',
+                    'Description': raw_response.get('description')
+                }
+            } for hash_name in FILE_HASHES if raw_response.get(hash_name)]
+        human_readable = tableToMarkdown(title, context_entry, removeNull=True)
+        return human_readable, context, raw_response
+    else:
+        return f'{INTEGRATION_NAME} - No results found for file hash: [{file_hash}', {}, raw_response
+
+
+def search_domain(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    """Gets a job from the API. Used mostly for polling playbook
+    """
+    url = args.get('domain', '')
+    raw_response = client.get_domain_request(url)
+    results = raw_response.get('result')
+    if results:
+        result = results[0]
+        title = f'{INTEGRATION_NAME} - Analysis results for domain: {url}'
+        context_entry = build_context(result, 'Domain')
+        # Building a score for DBot
+        score = client.calculate_dbot_score(result.get('severity'))
+        dbot_entry = build_dbot_entry(url, 'domain', INTEGRATION_NAME, score, result.get('description'))
+        context = {
+            f'{INTEGRATION_CONTEXT_NAME}(val.ID && val.ID === obj.ID)': context_entry
+        }
+        context.update(dbot_entry)
+        human_readable = tableToMarkdown(title, context_entry, removeNull=True)
+        return human_readable, context, raw_response
+    else:
+        return f'{INTEGRATION_NAME} - No results found for Domain: {url}', {}, raw_response
+
+
+def test_module(client: Client, *_) -> Tuple[str, Dict, Dict]:
+    raw_response = client.test_module_request()
+    if raw_response.get('version'):
+        return 'ok', {}, raw_response
+    raise DemistoException(f'Test module failed\nraw_response: {raw_response}')
+
+
+''' COMMANDS MANAGER / SWITCH PANEL '''
+
+
+def main():  # pragma: no cover
+    params = demisto.params()
+    base_url = params.get('url', '').rstrip('/') + '/api/v2'
+    verify = not params.get('insecure', False)
+    proxy: Optional[bool] = params.get('proxy')
```

Comment:
you are handling it different than in the authentication template, lets be consistent